Automattic/wp-calypso
# Enable VideoPress Onboarding trial for videomaker.

Related to https://github.com/Automattic/greenhouse/issues/1958

## Proposed Changes

* Enable videomaker trial from the VideoPress flow in all other envs, except test.
* Run e2e tests locally with `CALYPSO_BASE_URL=http://calypso.localhost:3000 yarn workspace wp-e2e-tests test -- specs/onboarding/signup__videopress-tailored.ts`.

## Testing Instructions



* Check that you get the trial flow in the container and locally. The flow starts from `/setup/videopress`

## Pre-merge Checklist
